configurations/conf.js

```diff
@@ -0,0 +1,7 @@
+exports.config = {
+    framework: "jasmine",
+    specs: ["../test/module_6/protractorTask.spec.js"],
+    capabilities: {
+        browserName: "firefox"
+    }
```

Comment:
Please set screen size inside onPrepare field of config

Reply:
Set screen size